src/refactor_nrepl/ns/libspec_whitelist.clj

```diff
@@ -0,0 +1,47 @@
+(ns refactor-nrepl.ns.libspec-whitelist
+  (:require
+   [clojure.java.io :as io]
+   [refactor-nrepl.config :as config])
+  (:import
+   (java.util.regex Pattern)))
+
+(defn- libspec-whitelist* []
+  (let [kondo-file (io/file ".clj-kondo" "config.edn")
+        exclude (when (.exists kondo-file)
+                  (try
+                    (-> kondo-file slurp read-string :linters :unused-namespace :exclude)
+                    (catch Exception e
+                      (when (System/getenv "CI")
+                        (throw e)))))]
+    (->> exclude
+         (mapv (fn [entry]
+                 (re-pattern (str "^" (Pattern/quote (str entry)) "$"))))
```

Comment:
I shouldn't Pattern/quote this if it's already a regex-like string. Official example:

```clj
{:linters {:unused-namespace {:exclude [".*\\.specs$"]}}}
```